record/record.app.js

```diff
@@ -57,7 +58,59 @@
         context.pageId = MathUtils.uuid();
 
         ERMrest.appLinkFn(UriUtils.appTagToURL);
-
+        /* @desc getPageSize(obj) returms page size of the obj.           */
```

Comment:
This isn't being formatted like a jdoc because there isn't an extra `*`. i.e. `/** @desc ...`